rest/route/ip_address.go

```diff
@@ -0,0 +1,58 @@
+package route
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/evergreen-ci/evergreen/model/host"
+	"github.com/evergreen-ci/evergreen/rest/data"
+	"github.com/evergreen-ci/gimlet"
+	"github.com/pkg/errors"
+)
+
+////////////////////////////////////////////////////////////////////////
+//
+// GET /rest/v2/hosts/ip_address/{ip_address}
+
+type addressGetHandler struct {
```

Comment:
nit: Can we maybe make this name a bit more specific? Maybe ipAddress or hostIpAddress instead of address?

Reply:
done